python/numeric-python.ipynb

```diff
@@ -61,9 +61,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "- Grunddatentyp von Numpy: das Array\n",
-    "- Kann man sich als effizientere Liste vorstellen\n",
-    "- Idee von Numpy: Man kann ein Array ähnlich wie eine Zahl verwenden. Operationen werden dann auf allen Elementen ausgeführt\n",
+    "- Grunddatentyp von Numpy ist das **Array**\n",
+    "- Arrays kann man sich als effizientere Listen vorstellen\n",
```

Comment:
Habe ein bissche Probleme mit dieser Formulierung. Was bedeutet effizienter? Es gibt Dinge bei denen sind Listen wesentlich effizienter als arrays (appenden).

Also vielleicht: `Numpy arrays speicher Werte *eines* Datentyps in einem zusammenhängendem Speicherbereich, dadurch sind mathemathische Operationen auf allen Werten eines arrays effizienter?

Reply:
Ja, stimme ich dir völlig zu. Gleicher Fall wie oben. Das stand da vorher so sloppy und ich war mir nicht sicher warum.

Reply:
Würdest du sagen, dass `structured arrays` es Wert sind über das `eines` zu diskutieren? Oder ist das so irrelevant, dass es hier nicht erwähnt werden muss?

Reply:
Ah, guter Punkt. Ich würde hier oben das array und eines stehen lassen und weiter unten wo wir bei der io geschichte structured arrays einführen das da kurz diskutieren. 

Da steht was von das structure arrays quasi dicts von arrays sind, was auch nicht wirklich stimmt, abseits davon das man getitem mit nem spaltennamen machen kann